src/lib/core.js

```diff
@@ -147,7 +147,7 @@ class Core extends EventEmitter {
       destination_address: query.destinationAddress,
       destination_amount: query.sourceAmount === undefined ? hop.finalAmount : undefined,
       source_expiry_duration: sourceExpiryDuration
-        ? (sourceExpiryDuration - hop.minMessageWindow)
+        ? (sourceExpiryDuration - headHop.minMessageWindow)
```

Comment:
The previous version was incorrect -- `hop.minMessageWindow` is the window for the entire quote, `headHop.minMessageWindow` is the window of the first step.

It helps to look at it as: `hop == headHop + tailQuote`, so to find the `sourceExpiryDuration` of the `tailQuote`, you need `sourceExpiryDuration - headHop.minMessageWindow`.